### PR TITLE
created homepage

### PR DESCRIPTION
**Pull request by sagargse**
Created Home Page of ecommerce.
Do check component folder for every component.
- Carousel Component
- Navbar Component
- Footer Component
Please give valuable suggestions and feedbacks
image

Preview Link - https://deploy-preview-1--ecommerce-kidszilla.netlify.app/

### CODE REVIEW

**Review by khushijohri2001 — APPROVED** [review]
UI is funky and cool!!
Code is clean
Good to go for merge

**Comment by sagargse** [comment]
> UI is funky and cool!! Code is clean Good to go for merge

okay! thanks

File: src/components/carousel/Carousel.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+export const Carousel = () => {
+    const [categories, setCategories] = useState([]);
+
+    const fetchCategories = async () => {
+        try {
+            const response = await axios.get('/api/categories');
+            setCategories(response.data.categories);
+            console.log(response.data.categories);
+        } catch (error) {
+            console.log(error);
```
Comment (Shailseen) [inline src/components/carousel/Carousel.jsx:12]: can you make an error page with UI ?

**Review by Shailseen — APPROVED** [review]
if you can make changes of the comment part then go for it. otherwise looks all fine for me.

File: src/components/carousel/Carousel.jsx (hunk 1 above)
Comment (sagargse) [reply src/components/carousel/Carousel.jsx:12]: okay!

**Pull request merged**